app/controllers/close_contacts_controller.rb

```diff
@@ -54,4 +54,22 @@ def update
                                created_by: current_user.email,
                                comment: "User edited a close contact (ID: #{cc.id}).")
   end
+
+  # Delete an existing close contact record
+  def destroy
+    redirect_to root_url && return unless current_user.can_create_patient_close_contacts?
+    cc = current_user.get_patient(params.permit(:patient_id)[:patient_id])&.close_contacts&.find_by(id: params.permit(:id)[:id])
+    redirect_to root_url && return if cc.nil?
+    cc.destroy
+    if cc.destroyed?
+      reason = params.permit(:delete_reason)[:delete_reason]
+      History.close_contact_edit(patient: params.permit(:patient_id)[:patient_id],
```

Comment:
might want to check with the PO team on the way we handle/display blank fields in history

<img width="1627" alt="Screen Shot 2021-05-18 at 10 32 52 AM" src="https://user-images.githubusercontent.com/9956561/118670367-8b17d500-b7c4-11eb-918f-72b7b26133bb.png">